src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -162,6 +162,12 @@ export function createTransaction(
         ? await apiRequest(route, options)
         : await localVet360.createTransaction();
 
+      if (transaction?.errors) {
+        const error = new Error();
```

Comment:
nit: might be worth adding a name for this error, like `new Error('Transaction creation failure')` just to make the debugging experience a tiny bit nicer?

Reply:
I have made a note of this and will make sure to make this change in a future PR I push up.